homeassistant/components/wiser/sensor.py

```diff
@@ -0,0 +1,579 @@
+"""
```

Comment:
Please limits the initial PR to a single platform, as per:

<https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible>

Reply:
Responded to this comment in the same comment for switch.py platform